react/src/pages/datasets/DatasetTable.tsx

```diff
@@ -131,6 +133,36 @@ export default function DatasetTable() {
                         }
                     }
                 }}
+                cellEditable={{
```

Comment:
Did you forget to add `emptyValue: '<empty>'` to the editable columns or you did that on purpose?

Reply:
My mistake. I'll fix that shortly.